src/CMakeLists.txt

```diff
@@ -748,6 +748,8 @@ if (NOT WITH_SYSTEM_ROCKSDB)
     list(APPEND ROCKSDB_CMAKE_ARGS -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER})
   endif(WITH_CCACHE AND CCACHE_FOUND)
 
+  # Quick fix for lack of SSE4.2-accelerated CRC32 calculation in RocksDB
+  list(APPEND ROCKSDB_CMAKE_ARGS -DCMAKE_CXX_FLAGS=-msse4.2)
```

Comment:
Sure. Thanks for quick review and pointing out the right way! :-) The patch is definitely awkward and I will try to rework it according to your suggestions.

However, what is currently the most interesting thing IMO is the impact on `kv_sync_thread`'s performance. I hope that even this silly patch might allow to start verification (need to check whether the `Slow_CRC32` isn't somehow restricted to my machines only) or at least bring some attention to this area.